editor.planx.uk/src/pages/Team/index.tsx

```diff
@@ -404,8 +333,23 @@ const Team: React.FC = () => {
     fetchFlows();
   }, [fetchFlows]);
 
+  const { data: templateData } = useQuery<{ flows: TemplateOverview[] }>(gql`
+    query GetTemplates {
+      flows(where: { is_template: { _eq: true } }) {
+        id
+        slug
+        name
+      }
+    }
+  `);
+
   const teamHasFlows = flows && Boolean(flows.length);
   const showAddFlowButton = teamHasFlows && canUserEditTeam(slug);
+  const showAddTemplateButton =
```

Comment:
This is a bit convoluted but will get simplified when there are always templates and the feature flag is removed.